meteor/server/DatabaseCaches.ts

```diff
@@ -279,6 +302,24 @@ export class CacheForRundownPlaylist extends CacheForStudioBase {
 		return super.deferAfterSave(fcn)
 	}
 }
+/** A read-only copy of CacheForRundownPlaylist */
+export interface ReadOnlyCacheForRundownPlaylist extends ReadOnlyCacheForStudioBase {
```

Comment:
If you wanted to have some fun with typescript instead of hand writing the ReadOnlyCacheFor* interfaces, you could do something like (stolen from my cache improvment branch :wink: )
The benefit of using a generic adapter is it will reduce the risk of error when editing the interfaces

```
export type ReadOnlyCacheInner<T> = T extends DbCacheWriteCollection<infer A, infer B>
	? DbCacheReadCollection<A, B>
	: T extends DbCacheWriteObject<infer A, infer B>
	? DbCacheReadObject<A, B>
	: T extends DbCacheWriteOptionalObject<infer A, infer B>
	? DbCacheReadObject<A, B, true>
	: T
export type ReadOnlyCache<T extends Cache> = { [K in keyof T]: ReadOnlyCacheInner<T[K]> }
```